src/main/java/build/buildfarm/instance/shard/RedisShardBackplane.java

```diff
@@ -1497,16 +1497,16 @@ public boolean canPrequeue() throws IOException {
   @SuppressWarnings("ConstantConditions")
   @Override
   public BackplaneStatus backplaneStatus() throws IOException {
-    BackplaneStatus.Builder builder = BackplaneStatus.newBuilder();
     Set<String> executeWorkers = getExecuteWorkers();
     Set<String> storageWorkers = getStorageWorkers();
-    builder.addAllActiveExecuteWorkers(executeWorkers);
-    builder.addAllActiveStorageWorkers(storageWorkers);
-    builder.addAllActiveWorkers(Sets.union(executeWorkers, storageWorkers));
-    builder.setDispatchedSize(client.call(jedis -> state.dispatchedOperations.size(jedis)));
-    builder.setOperationQueue(state.operationQueue.status(client.call(jedis -> jedis)));
-    builder.setPrequeue(state.prequeue.status(client.call(jedis -> jedis)));
-    return builder.build();
+    return BackplaneStatus.newBuilder()
+        .addAllActiveExecuteWorkers(executeWorkers)
+        .addAllActiveStorageWorkers(storageWorkers)
+        .addAllActiveWorkers(Sets.union(executeWorkers, storageWorkers))
+        .setDispatchedSize(client.call(jedis -> state.dispatchedOperations.size(jedis)))
+        .setOperationQueue(state.operationQueue.status(client.call(jedis -> jedis)))
```

Comment:
Apropos of nothing, but... what? This is yielding the jedis instance, that is returned to the pool - this is asking for a desync failure...
I know it's not related to your change, but I cannot believe this was in there.
This needs to be `client.call(jedis -> state.operationQueue.status(jedis))` and likewise for the following line with prequeue.

Reply:
OK if I make that change in a separate PR?

Reply:
Moved to https://github.com/bazelbuild/bazel-buildfarm/pull/1691